Cornelica/scenario/010.txt

```diff
@@ -182,45 +182,45 @@
 
 @huki b=7 id=12
 @bs f=リリィ@困惑 op=困惑 body=-1 face=0 x=350 y=50 t=200
-[Lily] "... After all ... ... when I was hugged a little while ago ...."
+[Lily]"...After all...when you was hugged a little while ago...."
```

Comment:
I think `when you were hugged` makes more sense here.

Reply:
If my old prof see this.... :D that's why is better to doublecheck my english